templates/shared_services/firewall/porter.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: tre-shared-service-firewall
-version: 0.6.0
+version: 0.6.1
```

Comment:
@tanya-borisova why would one upgrade to the newer firewall versions in the first place and run into this issue?
Iirc the changes in the recent versions are exactly that so a way to avoid the issue is not to upgrade the firewall. You can resolve this after considering.

Reply:
@tamirkamara  You are correct that if firewall isn't upgraded, customers don't run into this issue. Currently, in the CHANGELOG, customers are advised to upgrade firewall, at which point, if they have any old bundles still deployed, these will stop working. 

We have discussed two options with the team and others, for the way forward with the release: 

Option 1 is what you suggest. We make the release with code changes as-is, but we advise customers to not upgrade firewall until all their bundles are upgraded (to the ones that are built with the new mixin).

Option 2. We make a code change before we do the release, which would temporarily revert blocking of hashicorp domains in firewall. This means that customers have time to upgrade their bundles to use the new mixin, which is only available in this release. 

We think that doing option 2 provides a better upgrading strategy for customers, as they can take time to upgrade their bundles and be confident everything is upgraded, before they upgrade the firewall in the future. It also means we have a way to release any other change or fix to firewall, should the need arise, before customers finished upgrading their bundles.